src/app/Layout/Layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { Outlet } from 'react-router-dom';
+import { Navbar } from './Navbar';
+import { Sidebar } from './Sidebar';
+import { Footer } from './Footer';
+
+export const Layout = () => {
+  return (
+    <div>
+      <Navbar />
+      <Sidebar />
+      <Outlet />
+      <Footer />
+    </div>
+  );
+};
```

Comment:
In this Layout you should have the structure and all placeholders for now. You don't have to pre-prepare the components as probably not all of them will be needed in the way you thought. Keep it simple until you know you have to create a new component.